include/fiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -543,16 +577,13 @@ class operational_domain_impl
     [[nodiscard]] operational_domain<parameter_point, operational_status>
     contour_tracing(const std::size_t samples) noexcept
     {
+        assert(num_dimensions == 2 && "Contour tracing is only supported for two dimensions");
```

Comment:
@marcelwa Do we know the behavior if a user uses contour tracing with ``num_dimensions=3`` in release-mode/python? or is it rather an undefinied behavior?

Reply:
Ah, I have seen that an error thrown. Great!